ci.hocon

```diff
@@ -227,27 +228,42 @@ truffleruby-cexts: {
 gate-caps: {
   capabilities: [linux, amd64]
   targets: [gate, post-push]
+  environment: {
+    GITHUB_STATUS: true
+  }
 }
 
 gate-caps-darwin: {
   capabilities: [darwin, amd64]
   targets: [gate, post-push]
+  environment: {
+    GITHUB_STATUS: true
+  }
 }
 
 gate-caps-solaris: {
   capabilities: [solaris, sparcv9]
   targets: [gate, post-push]
+  environment: {
+    GITHUB_STATUS: true
+  }
 }
 
 daily-gate-caps: {
   capabilities: [linux, amd64]
   targets: [gate, post-push, daily]
```

Comment:
This seems a weird one, caps should be either post-push or daily but never both.
I think we should just remove this one and replace with `gate-caps` cc @chrisseaton 
But in another PR probably.